src/components/Sidebar/FooterMenu.react.js

```diff
@@ -1,69 +0,0 @@
-/*
- * Copyright (c) 2016-present, Parse, LLC
- * All rights reserved.
- *
- * This source code is licensed under the license found in the LICENSE file in
- * the root directory of this source tree.
- */
-import Icon     from 'components/Icon/Icon.react';
-import Popover  from 'components/Popover/Popover.react';
-import Position from 'lib/Position';
-import React    from 'react';
-import styles   from 'components/Sidebar/Sidebar.scss';
-
-let mountPath = window.PARSE_DASHBOARD_PATH;
-
-export default class FooterMenu extends React.Component {
-  constructor() {
-    super();
-
-    this.state = {
-      show: false,
-      position: null,
-    };
-    this.moreRef = React.createRef();
-  }
-
-  toggle() {
-    let pos = Position.inWindow(this.moreRef.current);
-    pos.x += 24;
-    this.setState({
-      show: true,
-      position: pos
-    });
-  }
-
-  render() {
-    if (this.props.isCollapsed) {
-      return (
-        <div className={styles.more}>
-          <Icon height={24} width={24} name='ellipses' />
-        </div>
-      );
-    }
-
-    let content = null;
-    if (this.state.show) {
-      content = (
-        <Popover
-          fixed={true}
-          position={this.state.position}
-          onExternalClick={() => this.setState({ show: false })}>
-          <div className={styles.popup}>
-            <a href={`${mountPath}logout`}>Log Out <span className={styles.emoji}>👋</span></a>
```

Comment:
You removed the whole footer, but that also removes the "Log Out" menu.

![image](https://user-images.githubusercontent.com/5673677/177033661-d67847ac-d51e-4a12-8c9e-c8677a215e60.png)

Reply:
Oh, that was a good one 🫢
<img width="323" alt="Screen Shot 2022-07-03 at 15 00 06" src="https://user-images.githubusercontent.com/64530191/177043469-b89e13fb-e363-40d4-803d-98bef9131e53.png">